ironfish/src/rpc/routes/wallet/getNotes.ts

```diff
@@ -48,8 +53,12 @@ router.register<typeof GetAccountNotesStreamRequestSchema, GetAccountNotesStream
       const transaction = await account.getTransaction(transactionHash)
       Assert.isNotUndefined(transaction)
 
+      const asset = await getAssetById(note.assetIdentifier(), node.chain)
+
       request.stream({
         amount: note.value().toString(),
```

Comment:
This is an existing bug I just noticed from the old code. This should be `CurrencyUtils.encode(note.value())`

Reply:
I'll fix it in another PR